crates/oxc_linter/src/rules/eslint/eq_eq_eq.rs

```diff
@@ -91,5 +127,17 @@ fn test() {
         ("value == undefined", None),
     ];
 
-    Tester::new(EqEqEq::NAME, pass, fail).test_and_snapshot();
+    let fix =
+        vec![("null==null", "null===null", None), 
```

Comment:
we don't have span in operator, is that better to add fixed space like `null === null` ?